mobile/lib/ui/viewer/actions/file_selection_actions_widget.dart

```diff
@@ -316,6 +320,15 @@ class _FileSelectionActionsWidgetState
         ),
       );
     }
+    if (widget.type != GalleryType.sharedPublicCollection) {
+      items.add(
+        SelectionActionButton(
+          icon: Icons.movie_creation_sharp,
+          labelText: "Video memory",
+          onTap: _onCreateVideoMemoryClicked,
+        ),
+      );
```

Comment:
Can you put it behind feature flag for internal users & rename the label text to `(i) Video Memory`